docs/icicle/integrations.md

```diff
@@ -23,17 +23,38 @@ The first approach for GPU-accelerating your Prover with ICICLE is quick to impl
 
 A full GPU replacement means performing the entire ZK proof on the GPU. This approach will reduce latency to a minimum and requires you to change the way you implement the protocol to be more GPU friendly. This approach will take full advantage of GPU acceleration. Redesigning your prover this way may take more engineering effort but we promise you that its worth it!
 
-## Using ICICLE integrations
+## Using ICICLE integrated provers
 
 Here we cover how a developer can run existing circuits on ICICLE integrated provers.
 
 ### Gnark
 
 [Gnark](https://github.com/Consensys/gnark) officially supports GPU proving with ICICLE. Currently only Groth16 on curve `BN254` is supported. This means that if you are currently using Gnark to write your circuits you can enjoy GPU acceleration without making many changes.
 
-To use GPUs, add the `icicle` buildtag to your build/run commands, e.g. `go run -tags=icicle main.go.`
+:::info
+Currently ICICLE has been merged to Gnark [master branch](https://github.com/Consensys/gnark), however a the [latest release](https://github.com/Consensys/gnark/releases/tag/v0.9.1) is from October 2023.
+:::
 
-You must also switch to the ICICLE backend, WithIcicleAcceleration backend ProverOption:
+Make sure your golang circuit project has `gnark` as a dependency. And make sure that you are using the master branch for now.
+
+```
+go get 	github.com/consensys/gnark@master
+```
+
+You should see two indirect dependencies added.
+
+```
+...
+	github.com/ingonyama-zk/icicle v0.1.0 // indirect
+	github.com/ingonyama-zk/iciclegnark v0.1.1 // indirect
+...
+```
+
+:::info
+As you may notice we are using ICICLE v0.1 here since golang bindings are only support in ICICLE v0.1 for the time being.
+:::
```

Comment:
Here as well, if we care

Reply:
ok i added the recommended docusarus prettier config